proto/regen/ecocredit/v1alpha1/query.proto

```diff
@@ -58,6 +58,51 @@ service Query {
     option (google.api.http).get =
         "/regen/ecocredit/v1alpha1/params";
   }
+
+  // SellOrder queries a sell order by its ID
+  rpc SellOrder(QuerySellOrderRequest) returns (QuerySellOrderResponse) {
+    option (google.api.http).get = "/regen/ecocredit/v1alpha1/sell-orders/id/{sell_order_id}";
+  }
+
+  // SellOrders queries a paginated list of all sell orders
+  rpc SellOrders(QuerySellOrdersRequest) returns (QuerySellOrdersResponse) {
+    option (google.api.http).get = "/regen/ecocredit/v1alpha1/sell-orders";
+  }
+
+  // SellOrdersByDenom queries a paginated list of all sell orders of a specific denom
+  rpc SellOrdersByDenom(QuerySellOrdersByDenomRequest) returns (QuerySellOrdersByDenomResponse) {
+    option (google.api.http).get = "/regen/ecocredit/v1alpha1/sell-orders/denom/{denom}";
+  }
```

Comment:
`denom` might be a bit ambiguous and maybe we should explicitly call this `batch_denom` (the denom for the credit batch, not the denom for the ask price).